### PR TITLE
Add core weekly 47

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

File: news/_posts/core-weekly/2020-11-23-coreweekly-47-2020.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+### PHP Developer Tools
+* [#39](https://github.com/PrestaShop/php-dev-tools/pull/39): phpstan config : add  `_PS_MODE_DEV_` to dynamicConstantNames. Thank you [@SebSept](https://github.com/SebSept)
+
+
+### Changes in developer documentation
+* [#801](https://github.com/PrestaShop/docs/pull/801): More informations for maintainers : red flags for code reviews, BC breaks, what it means to approve a PR, by [@matks](https://github.com/matks)
+* [#782](https://github.com/PrestaShop/docs/pull/782): Add tinymce config extend documentation, by [@NeOMakinG](https://github.com/NeOMakinG)
+
+
+## Where to start contributing?
```
Comment (matks) [inline news/_posts/core-weekly/2020-11-23-coreweekly-47-2020.md:169]: @LouiseBonnard 🎁

Comment (LouiseBonnard) [reply news/_posts/core-weekly/2020-11-23-coreweekly-47-2020.md:169]: 😍

Comment (LouiseBonnard) [reply news/_posts/core-weekly/2020-11-23-coreweekly-47-2020.md:169]: So nice, thanks a million! Let me know if you need a hand while waiting for it to be automatized, I can pick one every week and write a small introduction. :-)

File: news/_posts/core-weekly/2020-11-23-coreweekly-47-2020.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+Thanks to the contributors who tested the Release Candidate, a few bugs have been identified in the build that was published [three weeks ago](https://build.prestashop.com/news/prestashop-1-7-7-0-rc-release/).
+
+These bugs have been fixed, so the road is now open to deliver PrestaShop 1.7.7.0 stable release. A new build has been achieved and delivered to QA team to run the tests campaign before publishing it!
```
Comment (eternoendless) [inline news/_posts/core-weekly/2020-11-23-coreweekly-47-2020.md]:
```suggestion
These bugs have been fixed, so the road is now open to deliver PrestaShop 1.7.7.0 stable release. A new build has been created and delivered to QA team to run the tests campaign. Once it's verified, if no blocking bug is found, we will be able to release it!
```

File: news/_posts/core-weekly/2020-11-23-coreweekly-47-2020.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 47 of 2020"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2020-11-23
```
Comment (matks) [inline news/_posts/core-weekly/2020-11-23-coreweekly-47-2020.md]:
```suggestion
date:   2020-11-24
```

File: news/_posts/core-weekly/2020-11-23-coreweekly-47-2020.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+* [#172](https://github.com/PrestaShop/prestashop-specs/pull/172): Edit Shop Parameters > Search, by [@LouiseBonnard](https://github.com/LouiseBonnard)
+
+
+### Auto Upgrade module
+* [#352](https://github.com/PrestaShop/autoupgrade/pull/352): Remove VSCode config files, by [@matks](https://github.com/matks)
+* [#351](https://github.com/PrestaShop/autoupgrade/pull/351): Add travis badge status, by [@matks](https://github.com/matks)
+
+
+### Webservices PHP Client
+* [#73](https://github.com/PrestaShop/PrestaShop-webservice-lib/pull/73): Remove duplicate @throws in PHPDoc. Thank you [@davidglezz](https://github.com/davidglezz)
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-11-23-coreweekly-47-2020.md]:
```suggestion
* [#73](https://github.com/PrestaShop/PrestaShop-webservice-lib/pull/73): Remove duplicate `@throws` in PHPDoc. Thank you [@davidglezz](https://github.com/davidglezz)
```

**Review by Progi1984 — APPROVED** [review]

**Comment by matks** [comment]
Thank you for review!

**Pull request merged**